examples/shared_vpc/main.tf

```
@@ -38,7 +38,7 @@ module "host-project" {
  *****************************************/
 module "vpc" {
   source  = "terraform-google-modules/network/google"
-  version = "~> 2.5.0"
+  version = ">= 4.0.1, < 5.0"
```

Lets pin to latest major for other modules 
```suggestion
  version = "~> 4.1"
```

---

OK

---

Done